db/migrate/20230815195144_add_ref_users_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddRefUsersToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :author, null: false, foreign_key: true
+    add_index :comments, :author
```

Comment:
- [ ] The `foreign_key` attribute should refer to the users, even if the column id would be `author_id`

  ```
  ...foreign_key: { to_table: :users }
  ```

- [ ] Kindly remove this `add_index` line as `add_reference` above already adds the index.